Cloud Application Manager/Automating Deployments/script-box.md

```diff
@@ -21,11 +21,11 @@ When ready to test the configuration, click **Deploy**. Under the Deployment Box
 
 Don’t find what you need? Then click **Create a new deployment policy box**.
 
-The easiest way to understand script boxes is to build one. Follow this tutorial to build a simple box that says [Hello World](//www.ctl.io/guides/).
+The easiest way to understand script boxes is to build one. Follow this tutorial to build a simple box that says [Hello World](//www.ctl.io/knowledge-base/cloud-application-manager/getting-started/getting-started-with-hello-world/).
```

Comment:
Change link to `(../Getting Started/getting-started-with-hello-world.md)`

Reply:
Thanks Stephen.
I've committed your suggested changes.

Best regards